src/app/modules/notification/services/notification.service.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { NotificationService } from './notification.service';
+
+describe('NotificationService', () => {
+  let service: NotificationService;
+  beforeEach(() => {
+    service = new NotificationService();
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  describe('getDefaultOptions()', () => {
+    it('should return the default options', () => {
+      expect(service.getDefaultOptions()).toEqual({
+        duration: 5,
+        isDismissable: true,
+        isSelfClosing: true
+      });
+    });
+  });
+
+  describe('push()', () => {
+    it('should add a notification to the list with an id', () => {
```

Comment:
Yes I wanted to, but I don't know how to check that...

Reply:
There is two way to test that, use jasmine `spyOn` :
https://jasmine.github.io/2.0/introduction
Or with a timeout, you can wait the delay time and check if the notification has been removed successfully.

Timeouts are usually avoided because they slow down the execution time of the tests